src/modules/test/recipes/create-asb-json.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+template=SecurityBaselineTests.json
+
+(
+  cat "${template}-header";
+  ./mof-to-json.awk -v remediate=1 "${@}"
+  cat "${template}-mid";
+  ./mof-to-json.awk -v remediate=0 "${@}"
+  cat "${template}-footer";
+)
```

Comment:
Why split recipe in footer, header and mid parts? In general test recipes can contain any number of reload module sequences. If we'll add another to ASB's recipe, we'll then add a new mid2 etc? Please consider putting this back into a single test recipe JSON .